doc/code_examples/Tutorial_MapAlignment.cpp

```diff
@@ -62,8 +62,8 @@ int main(int argc, const char** argv)
   algorithm.align(toAlign, transformation);
 
   // store results
-  xml_file.store("Tutorial_MapAlignment_1.featureXML", reference);
-  xml_file.store("Tutorial_MapAlignment_2.featureXML", toAlign);
+  xml_file.storeFeatures("Tutorial_MapAlignment_1.featureXML", reference);
```

Comment:
As long as FileHandler can figure out the intended output type (ie the file extension) it will try to find the handler for that file type. If you call it with a file format that can't support features the call returns false.

Reply:
would throwing an exception not be better?
I imagine that this could lead to many undetected bugs otherwise.

Reply:
Hmm, good point. I will transition these to be exception based